unlock-js/src/__tests__/erc20.test.js

```diff
@@ -42,4 +42,11 @@ describe('erc20', () => {
       expect(balance).toEqual(1337)
     })
   })
+
+  describe('approveTransfer', () => {
+    // this functionality will need to be tested after we revist how we test
+    //this functionality. Ideally we should test the contents of the transaction
+    // that will be sent.
+    it.skip('should dispatch the appropriate transaction', async () => {})
```

Comment:
We need this one ;)

Reply:
@julien51 In the current state of the way the tests are structured I don't believe it is possible to create this in a way that actually provides confidence here. 

We can mock the network requests. mock the mining of a block, and finally, then return. Ideally being able to assert the content of the transaction or its hash should be sufficient here.  Let's discuss this in person